### PR TITLE
Update date and time article

### PR DESCRIPTION
**Pull request by araiford**
The Date and Time article was looking pretty funky, so I updated it with the latest, correct header styles and some hx columns.

### CODE REVIEW

File: _docs/style/date-time.md [hunk 1]
```diff
@@ -2,65 +2,104 @@
...
+<hr>
 
-# Timestamp
+<div class="hxRow">
+{% column left:"hxCol-4 hxCol-xs-12 hxCol-sm-12 hxCol-md-4 hxCol-lg-4" %}
 
+#### Timestamp
+-   Add seconds or milliseconds to the time only when that level specificity is
+    important
+{% endcolumn %}
```
Comment (nimbinatus) [inline _docs/style/date-time.md]: Add new line above the endcolumn tag

File: _docs/style/date-time.md [hunk 2]
```diff
@@ -2,65 +2,104 @@
...
 preview-image:
 resource: true
 ---
 
 {% include toc.html %}
 
-Always use human-friendly formatting when displaying date and/or time.
 
-# Date
+### Date
```
Comment (nimbinatus) [inline _docs/style/date-time.md]: Any reason to start at L3 vs L2?

File: _docs/style/date-time.md [hunk 3]
```diff
@@ -2,65 +2,104 @@
...
 
+<div class="hxRow">
+{% column left:"hxCol-4 hxCol-xs-12 hxCol-sm-12 hxCol-md-4 hxCol-lg-4" %}
+Always use human-readable, international-friendly formatting when displaying date and time.
```
Comment (nimbinatus) [inline _docs/style/date-time.md]: Add blank line above and below line 18 here to space out the column tags.

File: _docs/style/date-time.md [hunk 4]
```diff
@@ -2,65 +2,104 @@
...
 
 Timeago details length of time that has elapsed since an event (for example, a
 monitoring alarm or ticket update). This is helpful for orienting users to the
 timeline of events.
 
-## Best Practices
+#### Best Practices
 
 -   Display order:
     -   Under 1 minute: "just now"
```
Comment (technabors) [inline _docs/style/date-time.md]: Add a line break here.
### How it looks now
image
### With line break
image

**Comment by nimbinatus** [comment]
@araiford, sorry, I might have stomped all over this when I was finishing out #328 earlier this week...

File: _docs/style/date-time.md (hunk 2 above)
Comment (araiford) [reply _docs/style/date-time.md]: @nimbinatus yes -- it's because the page already has an H1 and H2.

**Comment by araiford** [comment]
@nimbinatus -- I made the other blank lines changes you mentioned.

**Pull request merged**